de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceHeaderImpl.java

```diff
@@ -341,4 +349,61 @@ public BigDecimal calculateTotalNetAmtFromLines()
 
 		return totalNetAmt;
 	}
+
+	@Override
+	public int getC_PaymentTerm_ID()
```

Comment:
InvoiceHeaderImpl it's a simple POJO. Please extract this functionality from here, intoduce a field for paymentTermId, set it...

Reply:
Ah.. ok
I saw the calculate method.. so.. from there
I will extract.